ebml/EbmlUnicodeString.h

```diff
@@ -51,7 +51,7 @@ class EBML_DLL_API UTFstring {
   std::size_t length() const {return UTF8string.length();}
 
   const std::string & GetUTF8() const {return UTF8string;}
-  void SetUTF8(const std::string &);
+  void SetUTF8(std::string_view);
```

Comment:
I understand a `string_view` won't copy the data, but passing an object by value still seems wrong. We should keep the `const &`.

Reply:
It’s recommended to pass a string_view by value and a string by const ref from what I’ve seen.

Reply:
That's correct. It's preferably to pass small structs by value (small as in the member values can be passed via registers). There are several resources out there that go into the details as of why that is. One I like is [this one](https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/).